api/helpers/passGenerator.js

```diff
@@ -1,10 +1,16 @@
 function passGenerator() {
   let password = '';
-  const str =
-    'ABCDEFGHIJKLMNOPQRSTUVWXYZ' + 'abcdefghijklmnopqrstuvwxyz0123456789@#$';
-  for (let i = 1; i <= 8; i++) {
-    const char = Math.floor(Math.random() * str.length + 1);
-    password += str.charAt(char);
+  const chars = [
```

Comment:
Much cleaner to place all the characters in one array and to include other special characters